ui/jobs/combo_tracker.ts

```diff
@@ -1,7 +1,23 @@
 import { kAbility, kComboBreakers } from './constants';
 
+type StartMap = {
+  [s: string]: {
+    id: string;
+    next: StartMap;
+  };
+};
+
+type ComboCallback = (id?: string) => void;
```

Comment:
Ah nice, I checked too and nothing is using `null`, just checking for `!id`.  So, this seems safe to change here.